seawatch_registration/urls.py

```diff
@@ -24,4 +25,5 @@
     path('questions/', QuestionView.as_view(), name='questions'),
     path('position/', RequestedPositionView.as_view(), name='add_requested_profile'),
     path('skills/', AddSkillsView.as_view(), name='add_skills'),
+    path('availability/', AvailabilityView.as_view(), name='availability')
```

Comment:
availabilities wäre hier als Pfad besser und laut neuer convention sollte der Name availability_list lauten. Die anderen Namen passe ich in meinem Refactoring Branch an ;)